tests/Search.test.js

```diff
@@ -48,28 +46,32 @@ describe('Tests for Search', function() {
         userId = 12345678910112;
         key = 'jjjehdnctsjyieoalskcjdhsnahsadndfnusdfsdfsd=';
 
-        executionPath = path.join(__dirname, 'library');
-        if (isWindowsOS) {
-            executionPath = path.join(__dirname, '..', 'library');
-        }
-        userConfigDir = path.join(__dirname, '..');
-
-        searchConfig = require('../js/search/searchConfig.js');
-        const { Search } = require('../js/search/search.js');
-        SearchApi = new Search(userId, key);
-        launchAgent = require('../js/search/utils/search-launchd.js');
-        realTimeIndexPath = path.join(userConfigDir, 'data', 'temp_realtime_index');
-        tempBatchPath = path.join(userConfigDir, 'data', 'temp_batch_indexes');
-        dataFolderPath = path.join(userConfigDir, 'data');
-        if (fs.existsSync(dataFolderPath)) {
-            deleteIndexFolders(dataFolderPath);
-        }
-        done();
+            executionPath = path.join(__dirname, 'library');
+            if (isWindowsOS) {
+                executionPath = path.join(__dirname, '..', 'library');
+            }
+            userConfigDir = path.join(__dirname, '..');
+            libSymphonySearch = require('../js/search/searchLibrary.js');
```

Comment:
we should have `require` statements at the beginning of the file.

Reply:
We will not be able to initialize the library until we specify the `userDir` and `executionDir`.